libs/spark/src/Unstable_AvatarButton/Unstable_AvatarButton.tsx

```diff
@@ -16,7 +16,7 @@ type _Unstable_AvatarButtonTypeMap<
   props: P &
     Pick<
       Unstable_AvatarProps,
-      'alt' | 'color' | 'imgProps' | 'sizes' | 'src' | 'srcSet'
+      'alt' | 'color' | 'imgProps' | 'size' | 'sizes' | 'src' | 'srcSet'
```

Comment:
Since we want to exclude `"small"`, re-define instead of picking this. Can copy whats in the Avatar file and just remove the small union.